src/js/editor/editor.js

```diff
@@ -293,7 +306,7 @@ class Editor {
     this.run(postEditor => {
       let nextPosition = postEditor.deleteAtPosition(position, direction, {unit});
       postEditor.setRange(nextPosition);
-    });
+    }, EDITOR_RUN_LAST_ACTION_STATE.DELETE);
```

Comment:
Yes, I agree with the private argument on run. Alternatives include:
- Having a more complicated return type that can include the undo grouping information.
- Duplicating Run which focuses on those two specific use cases.